### PR TITLE
feat(CDKeys): buttons, cover & refactor

### PR DESCRIPTION
**Pull request by DarkVillager**
## Description 

Added buttons, cover & refactored

## Acknowledgements
- [X] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [X] I linted the code by running `yarn format`
- [X] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

</details>

### CODE REVIEW

File: websites/C/CDKeys/presence.ts [hunk 1]
```diff
@@ -2,60 +2,86 @@ const presence = new Presence({
...
+		];
+		presenceData.details = "Viewing product";
+		presenceData.state = document.querySelector<HTMLMetaElement>(
 			"meta[property='og:title']"
 		).content;
-	} else if (page.includes("/pc")) {
+	} else if (series) {
+		presenceData.buttons = [
+			{
+				label: "View procuct series",
```
Comment (EncryptedDev) [inline websites/C/CDKeys/presence.ts:59]: Typo

File: websites/C/CDKeys/presence.ts [hunk 2]
```diff
@@ -2,60 +2,86 @@ const presence = new Presence({
...
-	else if (
-		document.querySelector("#maincontent > div.page-title-wrapper > h1 > span")
-	) {
-		presenceData.details = document.querySelector(
-			"#maincontent > div.page-title-wrapper > h1 > span"
-		).textContent;
+	} else if (game) {
+		presenceData.buttons = [
+			{
+				label: "View game",
```
Comment (EncryptedDev) [inline websites/C/CDKeys/presence.ts:31]: Buttons should use Title Case

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/C/CDKeys/presence.ts (hunk 2 above)
Comment (EncryptedDev) [reply websites/C/CDKeys/presence.ts]: This has not been resolved fully

Comment (DarkVillager) [reply websites/C/CDKeys/presence.ts]: > This has not been resolved fully

Oh i see

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**